cli/cliui/agent.go

```diff
@@ -400,4 +401,12 @@ func ConnDiagnostics(w io.Writer, d ConnDiags) {
 	if d.AgentNetcheck != nil && d.AgentNetcheck.NetInfo != nil && d.AgentNetcheck.NetInfo.MappingVariesByDestIP.EqualBool(true) {
 		_, _ = fmt.Fprint(w, "❗ Agent is potentially behind a hard NAT, as multiple endpoints were retrieved from different STUN servers\n")
 	}
+
+	if d.ClientIPIsAWS {
+		_, _ = fmt.Fprint(w, "❗ Client IP address is within an AWS range, and is therefore behind a hard NAT\n")
+	}
+
+	if d.AgentIPIsAWS {
+		_, _ = fmt.Fprint(w, "❗ Agent IP address is within an AWS range, and is therefore behind a hard NAT\n")
```

Comment:
This could either explain the hard NAT is responsible for impeding a direct connection, or include a link to further reading?